sdk/communication/Azure.Communication.CallingServer/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 - Support for Audio Only Recording. User can provide with content Type e.g Audio/AudioVideo, user can also provide with channel type e.g Mixed/UnMixed, and user can also provide format type e.g mp4, mp3.
+- Delete recording apis
```

Comment:
nit: Can we change this to "New apis added to delete recordings" or anything more If you want to add